basesandbox/src/execution/executee.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+
+use crate::ipc::semaphore::{Mode, Semaphore};
+use crate::ipc::Ipc;
+
+// Interface for the sandboxee written in Rust
+pub struct Context<T: Ipc> {
+    pub ipc: T,
+    pub semaphore: Semaphore,
+}
+
+pub fn start<T: Ipc>() -> Context<T> {
+    println!("{:?}", std::time::SystemTime::now());
```

Comment:
Printing the current time without any comments will not help users.

Reply:
Ah it was for the debug. I'll remove it.